api/python/provisioner/commands/__init__.py

```diff
@@ -112,6 +114,11 @@ class RunArgsPillarGet(RunArgsBase):
     local: bool = RunArgs.local
 
 
+@attr.s(auto_attribs=True)
+class RunArgsPillarDump(RunArgsBase):
+    local: bool = RunArgs.local
```

Comment:
As stated in EOS-16560 we need also an optional argument for path to be generated, default value is `/opt/seagate/cortx_configs/provisioner_cluster.json`